.circleci/config.yml

```diff
@@ -140,14 +140,8 @@ workflows:
   build-test-deploy:
     jobs:
 
-      - approve_preview_build:
```

Comment:
Removed because this gets updated so rarely seems mad to have to approve it before it'll even build to dev.